src/Features/CSharp/Portable/Completion/KeywordRecommenders/NameOfKeywordRecommender.cs

```diff
@@ -23,15 +23,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
                 context.IsAnyExpressionContext ||
                 context.IsStatementContext ||
                 context.IsGlobalStatementContext ||
-                IsAttributeArgumentContext(context) ||
                 context.LeftToken.IsInCastExpressionTypeWhereExpressionIsMissingOrInNextLine();
         }
-
-        private static bool IsAttributeArgumentContext(CSharpSyntaxContext context)
-        {
-            return
-                context.IsAnyExpressionContext &&
```

Comment:
At the point `IsAttributeArgumentContext` is called, we already know that `context.IsAnyExpressionContext` is false because the caller does `context.IsAnyExpressionContext || ... || IsAttributeArgumentContext(context)`